doc/source/parameters/cfd/interface_sharpening.rst

```diff
@@ -1,8 +1,9 @@
 Interface sharpening
 --------------------
 
-When running a simulation containing multiple fluids using the volume of fluid method (VOF), the interface between two fluids becomes more blurry after each time step due to diffusion. 
-To avoid that, this subsection includes parameters related to the interface sharpening. The default parameters are:
+When running a multiphase flow simulation using the volume of fluid method (VOF), the interface between two fluids becomes more and more blurry after each time step due to diffusion. 
```

Comment:
Is the diffusion numerical? If so I'd specify it so there's no ambiguity

Reply:
very relevant indeed!

Reply:
Yup it's purely numerical diffusion